scripts/utils/migrate/fetch-redirects.js

```diff
@@ -0,0 +1,35 @@
+const fetch = require('node-fetch');
+const logger = require('../logger');
+const { paginate } = require('./docs-api');
+
+require('dotenv').config();
+
+const fetchRedirects = async (nodeId) => {
+  const url = `https://docs-dev.newrelic.com/api/migration/redirects?id=${encodeURIComponent(
+    `node/${nodeId}`
+  )}`;
+
+  try {
+    const res = await fetch(url, {
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Phpshield-Key-Disable': process.env.ACQUIA_DEV_PHP_SHIELD_KEY,
+      },
```

Comment:
Nit: we can probably move these headers up into a constant rather than redefining them every time.

Reply:
after refactor, that method was removed because we weren't using it.